cvat/apps/consensus/models.py

```diff
@@ -16,10 +16,8 @@
 
 class ConsensusConflictType(str, Enum):
     NoMatchingItemError = "no_matching_item"
-    FailedAttrVotingError = "failed_attribute_voting"
     NoMatchingAnnError = "no_matching_annotation"
     AnnotationsTooCloseError = "annotation_too_close"
-    WrongGroupError = "wrong_group"
```

Comment:
Why these 2 error types were removed?

Reply:
The attribute voting error here corresponds to annotation attrs, like rotation. But as we discussed that the user only knows task attributes as attributes.
Also, these two errors were not useful also in the sense of the user, as by default, we should have the attrs from the merged annotation which still happens but was raising these errors due to `quorum`

Reply:
> The attribute voting error here corresponds to annotation attrs, like rotation. But as we discussed that the user only knows task attributes as attributes.

Rotation is included in attributes in datumaro representation of CVAT annotations along with user-defined attributes. I don't see why conflicts in user-defined attributes should be ignored.

How it's related to groups? Probably, you need to check if the resulting groups are valid, if they were present before merging.